app/src/main/java/de/tum/in/tumcampusapp/api/app/TUMCabeClient.java

```diff
@@ -355,10 +355,8 @@ public void getChatMemberByLrzId(String lrzId, Callback<ChatMember> callback) {
 
     // TICKET SALE
     // Getting event information
-    public List<Event> getEvents() throws IOException {
-        List<Event> list = service.getEvents().execute().body();
-
-        return list;
+    public void getEvents(Callback<List<Event>> cb) {
+        service.getEvents().enqueue(cb);
```

Comment:
Please consider renaming that method, since it doesn't get anything anymore, only only fetches it